docs/src/app/components/pages/components/divider.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import dividerCode from '!raw!material-ui/lib/divider';
+import CodeExample from '../../code-example/code-example';
+import PropTypeDescription from '../../PropTypeDescription';
+import DividerExampleForm from '../../Divider/ExampleForm';
+import dividerExampleFormCode from '!raw!../../Divider/ExampleForm';
+import DividerExampleList from '../../Divider/ExampleList';
+import dividerExampleListCode from '!raw!../../Divider/ExampleList';
+import DividerExampleMenu from '../../Divider/ExampleMenu';
+import dividerExampleMenuCode from '!raw!../../Divider/ExampleMenu';
+import MarkdownElement from '../../MarkdownElement';
+import dividerReadmeText from '../../Divider/README';
+
+export default class AppBarPage extends React.Component {
+
+  constructor(props) {
```

Comment:
Nope, in matter of fact, now that you mention it, I copied the `AppBarPage` as a stub for this so I could see how it worked and forgot to rename the class too. I just fixed this locally to be a simple stateless function like the example components.